src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -30,18 +29,49 @@
 public final class ReferenceConfidenceVariantContextMerger {
 
     private final GenotypeLikelihoodCalculators calculators;
-    private final VCFHeader vcfInputHeader;
+    private static VCFHeader vcfInputHeader = null;
     protected final VariantAnnotatorEngine annotatorEngine;
+    private final boolean doSomaticMerge;
+    protected boolean dropSomaticFilteringAnnotations;
     protected final OneShotLogger oneShotAnnotationLogger = new OneShotLogger(this.getClass());
     protected final OneShotLogger oneShotHeaderLineLogger = new OneShotLogger(this.getClass());
     protected final OneShotLogger AS_Warning = new OneShotLogger(this.getClass());
+    private static final List<String> SOMATIC_FILTERING_ANNOTATIONS = Arrays.asList(
+            GATKVCFConstants.MEDIAN_MAPPING_QUALITY_KEY,
+            GATKVCFConstants.MEDIAN_BASE_QUALITY_KEY,
+            GATKVCFConstants.MEDIAN_READ_POSITON_KEY,
+            GATKVCFConstants.MEDIAN_FRAGMENT_LENGTH_KEY,
+            GATKVCFConstants.STRAND_ARTIFACT_POSTERIOR_KEY,
+            GATKVCFConstants.STRAND_ARTIFACT_AF_KEY
+    );
+    List<String> SOMATIC_INFO_ANNOTATIONS_TO_MOVE = Arrays.asList(GATKVCFConstants.TUMOR_LOD_KEY);
+
+    private static final List<String> SOMATIC_FORMAT_ANNOTATIONS_TO_KEEP = Arrays.asList(
+            GATKVCFConstants.ORIGINAL_CONTIG_MISMATCH_KEY,
+            GATKVCFConstants.STRAND_ARTIFACT_AF_KEY,
+            GATKVCFConstants.STRAND_ARTIFACT_POSTERIOR_KEY,
+            GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_GT_KEY,
```

Comment:
Because it's a name that's now outdated.  I'm not sure what else to call it.  ASSEMBLY_BASED_CALLER_PHASING_GT_KEY just doesn't have a great ring to it.

Reply:
It's fine to leave it, I was just confused.